core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -376,8 +376,8 @@ <h2 ng-if="collectionPlaythrough.hasFinishedCollection()" class="oppia-page-head
           <div class="title-section" style="background-color: <[collectionSummary.thumbnail_bg_color]>; z-index: 1;">
             <img ng-src="<[collectionSummary.thumbnail_icon_url]>" class="thumbnail-image">
           </div>
-          <div class="title-section-mask" style="padding: 26px;">
-            <span style="font-size: 14px;">Hover over an icon to preview an exploration.</span>
+          <div class="title-section-mask objective oppia-activity-summary-tile-pre-hover-preview">
```

Comment:
Hello Sean, the .objective class is the class that formats the exploration card, since the aim is to make the text in the "blank" card have the same formatting as the text in the "exploration" card, there is no need writing code to do what the objective class is already doing, just inherit the class and the formating then becomes the same.